src/test/java/com/flightstats/hub/dao/aws/ClusterContentServiceTest.java

```diff
@@ -0,0 +1,96 @@
+package com.flightstats.hub.dao.aws;
+
+import com.flightstats.hub.cluster.LastContentPath;
+import com.flightstats.hub.dao.ChannelService;
+import com.flightstats.hub.dao.ContentDao;
+import com.flightstats.hub.model.*;
+import com.flightstats.hub.util.HubUtils;
+import com.flightstats.hub.util.TimeUtil;
+import lombok.SneakyThrows;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.Optional;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ClusterContentServiceTest {
+    @Mock private ClusterContentService ccs;
+    @Mock private ChannelService channelSvc;
+    @Mock private ContentDao mockSpokeWriteDao, mockSpokeReadDao;
+    @Mock private ContentDao mockS3SingleDao, mockS3LargeDao, mockS3BatchDao;
+    @Mock private S3WriteQueue s3WriteQueue;
+    @Mock private ChannelConfig channelConfig;
+    @Mock private LastContentPath lastContentPath;
+    @Mock private HubUtils hubUtils;
+
+    private Content content;
+    private ContentKey contentKey;
+    private String channelName;
+
+    @Before
+    public void initClusterContentService() {
+        channelName = "/testChannel";
+        when(channelSvc.getCachedChannelConfig(channelName)).thenReturn(Optional.of(channelConfig));
+        ccs = new ClusterContentService(channelSvc, mockSpokeWriteDao, mockSpokeReadDao, mockS3SingleDao, mockS3LargeDao, mockS3BatchDao, s3WriteQueue, lastContentPath, hubUtils);
+    }
+
+    @SneakyThrows
+    private void initSimpleContent(boolean isLarge) {
```

Comment:
I usually put private methods below public ones, though I do get that it's helpful to know what helper methods are doing before they're used.

Reply:
Sure thing.  I'll move this down.